<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>648</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>801</width>
      <height>581</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QTabWidget" name="tabWidget">
       <property name="statusTip">
        <string/>
       </property>
       <property name="tabShape">
        <enum>QTabWidget::TabShape::Rounded</enum>
       </property>
       <property name="currentIndex">
        <number>0</number>
       </property>
       <widget class="QWidget" name="Introduction">
        <attribute name="title">
         <string>Introduction</string>
        </attribute>
        <widget class="QTextBrowser" name="textBrowser">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>0</y>
           <width>791</width>
           <height>551</height>
          </rect>
         </property>
         <property name="html">
          <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;meta charset=&quot;utf-8&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
hr { height: 1px; border-width: 0; }
li.unchecked::marker { content: &quot;\2610&quot;; }
li.checked::marker { content: &quot;\2612&quot;; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Segoe UI'; font-size:9pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;Welcome to &lt;span style=&quot; color:#aaaaff;&quot;&gt;Plancksoft's &lt;/span&gt;&lt;span style=&quot; color:#ffaaff;&quot;&gt;Scruta&lt;/span&gt;&lt;span style=&quot; color:#aaaaff;&quot;&gt;Net&lt;/span&gt; installer.&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;Please click &lt;span style=&quot; color:#005500;&quot;&gt;next&lt;/span&gt; button to go to &lt;span style=&quot; color:#005500;&quot;&gt;next&lt;/span&gt; step.&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;Or you can click &lt;span style=&quot; color:#ff0000;&quot;&gt;back&lt;/span&gt; button to go &lt;span style=&quot; color:#ff0000;&quot;&gt;back&lt;/span&gt; a step.&lt;/p&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;br /&gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-weight:700;&quot;&gt;Thank you.&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
        </widget>
       </widget>
       <widget class="QWidget" name="Configuration">
        <attribute name="title">
         <string>Configuration</string>
        </attribute>
        <widget class="QTextEdit" name="txtInstallationPath">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>100</y>
           <width>691</width>
           <height>31</height>
          </rect>
         </property>
        </widget>
        <widget class="QLabel" name="lblInstallPath">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>70</y>
           <width>771</width>
           <height>20</height>
          </rect>
         </property>
         <property name="text">
          <string>Please choose the installation path:</string>
         </property>
        </widget>
        <widget class="QPushButton" name="browseButton">
         <property name="geometry">
          <rect>
           <x>710</x>
           <y>93</y>
           <width>75</width>
           <height>41</height>
          </rect>
         </property>
         <property name="text">
          <string>Browse</string>
         </property>
        </widget>
       </widget>
       <widget class="QWidget" name="tab">
        <attribute name="title">
         <string>Download</string>
        </attribute>
        <widget class="QProgressBar" name="progressBarDownload">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>220</y>
           <width>771</width>
           <height>23</height>
          </rect>
         </property>
         <property name="value">
          <number>0</number>
         </property>
        </widget>
        <widget class="QPushButton" name="startButton">
         <property name="geometry">
          <rect>
           <x>390</x>
           <y>350</y>
           <width>121</width>
           <height>31</height>
          </rect>
         </property>
         <property name="text">
          <string>Start Download</string>
         </property>
        </widget>
        <widget class="QPushButton" name="resumeButton">
         <property name="geometry">
          <rect>
           <x>520</x>
           <y>350</y>
           <width>121</width>
           <height>31</height>
          </rect>
         </property>
         <property name="text">
          <string>Pause Download</string>
         </property>
        </widget>
        <widget class="QPushButton" name="cancelButton">
         <property name="geometry">
          <rect>
           <x>640</x>
           <y>350</y>
           <width>121</width>
           <height>31</height>
          </rect>
         </property>
         <property name="text">
          <string>Cancel Download</string>
         </property>
        </widget>
        <widget class="QLabel" name="retryLabel">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>130</y>
           <width>771</width>
           <height>31</height>
          </rect>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
        <widget class="QLabel" name="speedLabel">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>290</y>
           <width>771</width>
           <height>31</height>
          </rect>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
        <widget class="QLabel" name="sizeLabel">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>250</y>
           <width>771</width>
           <height>31</height>
          </rect>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
        <widget class="QLabel" name="etaLabel">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>170</y>
           <width>781</width>
           <height>31</height>
          </rect>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </widget>
       <widget class="QWidget" name="Installation">
        <attribute name="title">
         <string>Installation</string>
        </attribute>
        <widget class="QProgressBar" name="progressBar">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>220</y>
           <width>771</width>
           <height>23</height>
          </rect>
         </property>
         <property name="value">
          <number>0</number>
         </property>
        </widget>
        <widget class="QLabel" name="labelTime">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>250</y>
           <width>771</width>
           <height>20</height>
          </rect>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
        <widget class="QLabel" name="lblInstallationStatus">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>190</y>
           <width>731</width>
           <height>16</height>
          </rect>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </widget>
       <widget class="QWidget" name="Finish">
        <attribute name="title">
         <string>Finish</string>
        </attribute>
        <widget class="QTextBrowser" name="textBrowser_2">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>0</y>
           <width>791</width>
           <height>551</height>
          </rect>
         </property>
         <property name="html">
          <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;meta charset=&quot;utf-8&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
hr { height: 1px; border-width: 0; }
li.unchecked::marker { content: &quot;\2610&quot;; }
li.checked::marker { content: &quot;\2612&quot;; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Segoe UI'; font-size:9pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;You have finished installing the ScrutaNet.&lt;/p&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;br /&gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-weight:700;&quot;&gt;Thank you.&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
        </widget>
        <widget class="QCheckBox" name="cbLaunch">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>220</y>
           <width>761</width>
           <height>20</height>
          </rect>
         </property>
         <property name="text">
          <string>Do you wish to launch ScrutaNet?</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </widget>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QPushButton" name="nextButton">
    <property name="geometry">
     <rect>
      <x>720</x>
      <y>580</y>
      <width>75</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Next</string>
    </property>
   </widget>
   <widget class="QPushButton" name="backButton">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>580</y>
      <width>75</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Back</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
